packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -255,7 +255,8 @@ export const getStyles = memoizeFunction((
     rootDisabled: getDisabledStyles(theme),
 
     rootError: {
-      borderColor: ComboBoxRootColorErrored
+      borderColor: ComboBoxRootColorErrored,
+      marginBottom: '5px'

```

Comment:
In the JavaScript styling, specifying a number rather than a string will default to the px suffix. Just fyi, this is fine as well.